### PR TITLE
[APPTS-9652] Update env urls to axway domains

### PR DESCRIPTION
**Pull request by pkclark**
### Deprecation Notice

This version removes availability of `Env.webeventurl` and `Env.cacheurl` as those services will be sunset.

### Adoption Warning

The changes to `Env.registryurl` and `Env.pubsuburl` precede those services' availability on the updated domains.  Please do not adopt this version if you've not replaced references to those services or provided overrides for their current domains.

### CODE REVIEW

File: lib/env.js [hunk 1]
```diff
@@ -4,21 +4,17 @@ const debug = require('debug')('Appc:sdk:env');
 
 const envs = {
 	Production: {
-		baseurl: 'https://platform.appcelerator.com',
```
Comment (pkclark) [inline lib/env.js]: @jareddlc the good stuff is in this file.

**Comment by pkclark** [comment]
@jareddlc any ideas on that nyc error in Jenkins land?

File: test/analytics.js [hunk 2]
```diff
@@ -6,12 +6,9 @@ const fs = require('fs');
 
 const bodyparser = require('body-parser');
 const express = require('express');
-const request = require('request');
 const should = require('should');
 const wrench = require('wrench');
 
-const helper = require('./lib/helper');
```
Comment (jareddlc) [inline test/analytics.js]: this require is needed as it loads the global config

File: test/app.js [hunk 3]
```diff
@@ -5,7 +5,6 @@ const path = require('path');
 const should = require('should');
 
 const Appc = require('../');
-const helper = require('./lib/helper');
```
Comment (jareddlc) [inline test/app.js]: @pkclark this require is needed as it loads the global config

Comment (pkclark) [reply test/app.js]: @jareddlc went back and forth on directly calling conf loader and then left it in a bad state in this comment.  Readding.  One sec....

Comment (jareddlc) [reply test/app.js]: @pkclark not sure if the others are needed as well

**Comment by pkclark** [comment]
@whitfin thanks for the cleanup. At this point, the rest of the issues aren't regressing on this change and the env-related tests pass, so merging and we'll circle back before tooling looks to adopt.

**Pull request merged**